lib/mayaUsd/render/pxrUsdMayaGL/sceneDelegate.cpp

```diff
@@ -155,8 +163,6 @@ PxrMayaHdSceneDelegate::PxrMayaHdSceneDelegate(
         renderIndex->InsertTask<HdxShadowTask>(this, _shadowTaskId);
         _ValueCache&        cache = _valueCacheMap[_shadowTaskId];
         HdxShadowTaskParams taskParams;
-        taskParams.camera = _cameraId;
-        taskParams.viewport = _viewport;
```

Comment:
These parameters were not used by the task at least as far back as USD 20.02 and most likely were never used.